packages/dataviews/src/dataviews-layouts/list/style.scss

```
@@ -16,25 +16,36 @@ div.dataviews-view-list {
 		}
 
 		.dataviews-view-list__item-actions {
-			flex: 0;
-			overflow: hidden;
+			display: flex;
+			width: max-content;
+			flex: 0 0 auto;
+			gap: $grid-unit-15 * 0.5; // 6px.
```

```suggestion
			gap: $grid-unit-05;
```

Nit: As you pointed out, the button was overflowing the width of the wrapper, so the `6px` was a false calculation and probably incorrect. How about using `4px` (`$grid-unit-05`) instead?